assets/js/door.js

```diff
@@ -45,12 +45,12 @@ export default function(socket) {
   const channel = socket.channel(`door:${doorId}`, {})
 
   channel.join()
-    .receive("ok", resp => {
-      channel.push("ring", {client_id: client.id})
+    .receive('ok', resp => {
+      channel.push('ring', {client_id: client.id})
     })
-    .receive("error", resp => { console.log("Unable to join", resp) })
+    .receive('error', resp => { console.log('Unable to join', resp) })
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)